docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -73,7 +73,7 @@ Transform                             Description
 
 In this example, the table is partitioned by month and further divided into 10 buckets based on a hash of the account number::
 
-    CREATE TABLE iceberg.testdb.sample_partitioned (
+    CREATE TABLE iceberg.testdb.customer_accounts (
```

Comment:
I think the rename might be okay for having a consistent running example and just as more descriptive naming for the table. @ebyhr do you see any concerns with it?

Reply:
I think both is fine, no concern. What we want to express here is about partitioned table, so the original name just looked natural to me. Then, let's fix commit message because it's not typo.